data/json/item_groups.json

```diff
@@ -9387,5 +9387,81 @@
       ["boxer_shorts", 30],
       ["briefs", 30]
     ]
+  },{
+    "type" : "item_group",
+    "id" : "army_personal_locker",
+    "items" : [
+      ["jumpsuit", 40],
+      ["pants_army", 70],
+      ["jacket_army", 70],
+      ["socks", 40],
+      ["winter_pants_army", 20],
+      ["winter_jacket_army", 20],
+      ["winter_gloves_army", 20],
```

Comment:
This came up recently: would the locker be more likely to contain "extras" of the gear the soldier is wearing, or gear in storage for some other time?  If extras, it should probably prefer winter gear as that's the season when the Cataclysm took place.  (In fact, we're right around that time, I believe!)  Otherwise, sure, this'll work.